### PR TITLE
Exercise 5

### PR DESCRIPTION
**Pull request by rysharprules**
Moved src files into folder as per best practices and simplicity hence minor changes to some *.py files.

### CODE REVIEW

**Comment by Jonesey13** [comment]
Designated Reviewer: Alex

File: README.md [hunk 1]
```diff
@@ -2,28 +2,6 @@
...
-
-PowerShell
-
-```bash
-$ ./setup --windows
-```
-
 ### Environment variables
 
 To run effectively, update the `.env` file with your application details:
```
Comment (Jonesey13) [inline README.md]: Note that the `.env` file needs to be copied from `.env.template` for this to work

Also `.env.template` still has the old value of `FLASK_APP` (`app` and not `todo_app/app`)

File: README.md [hunk 2]
```diff
@@ -185,3 +163,23 @@ may request access to complete setup of configured shares.
...
+### Containers
+
+The application can run with Docker.
+
+#### Development
+
+1. Create a docker image with name and tag targetted for `dev`: 
+   1. `docker build --target dev -t todo-app:dev .`
+2. Start the container with `docker-compose` 
+   1. `docker-compose up -d`
```
Comment (Jonesey13) [inline README.md:176]: Do we need to run this detached?

File: docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+version: "3.9"
+
+services:
+  todo_app:
+    container_name: todo_app
+    image: todo-app:dev
```
Comment (Jonesey13) [inline docker-compose.yaml:6]: Could you see how to make this compose file use the `Dockerfile` directly?

**Review by Jonesey13 — APPROVED** [review]
This all works well (apart from an incorrect `.env.template` value, see below). Please feel free to merge this PR in when ready 👍

File: docker-compose.yaml (hunk 3 above)
Comment (rysharprules) [reply docker-compose.yaml:6]: I could use the  `dockerfile: Dockerfile` attr under `build`?

File: README.md (hunk 2 above)
Comment (rysharprules) [reply README.md:176]: My thinking was this separates the compose execution from the container but I guess it's not needed

File: README.md (hunk 1 above)
Comment (rysharprules) [reply README.md]: Updated in 26bb5cb1c97a932218a772fde560fac452a02bfc

**Pull request merged**